### PR TITLE
docs: Fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes various typos I spotted inside the project.

### CODE REVIEW

**Comment by srebhan** [comment]
@deining can you please run `make docs` and fix the merge conflict!?!

File: plugins/inputs/jenkins/sample.conf [hunk 1]
```diff
@@ -46,7 +46,7 @@
   ## Empty this field will use default value 5
   # max_connections = 5
 
-  ## When set to true will add node labels as a comma-seperated tag. If none,
+  ## When set to true will add node labels as a comma-separated tag. If none,
   ## are found, then a tag with the value of 'none' is used. Finally, if a
-  ## lable contains a comma it is replaced with an underscore.
+  ## labal contains a comma it is replaced with an underscore.
```
Comment (Hipska) [inline plugins/inputs/jenkins/sample.conf:51]:
```suggestion
  ## label contains a comma it is replaced with an underscore.
```

**Review by Hipska — CHANGES_REQUESTED** [review]

**Review by powersj — APPROVED** [review]
Thanks! I've rebased, made one change, and run make docs.

**Review by srebhan — APPROVED** [review]
Thanks a lot for the fixes @deining! I do agree with Josh that we should ignore the readme linter for this PR...

**Comment by Hr0bar** [comment]
The fireboard plugin actually does 'return "Celcius"' as a value, so showing the correct spelling in docs is misleading. Would need to be fixed in the return statement too, in which case its a breaking change.

**Pull request merged**